examples/jsm/renderers/nodes/lights/LightContextNode.js

```diff
@@ -0,0 +1,39 @@
+import ContextNode from '../core/ContextNode.js';
+import { RE_Direct_BlinnPhong } from '../functions/BSDFunctions.js';
+
+class LightContextNode extends ContextNode {
+
+	constructor( node, material = null ) {
+
+		super( node );
+		
+		this.material = material;
+		
+	}
+
+	generate( builder, output ) {
+
+		// use an alternative material if exist
+		const material = this.material !== null ? this.material : builder.material;
+
+		let RE_Direct = null;
+		
+		if ( material.isMeshPhongMaterial === true ) {
```

Comment:
Any reasons why  you have decided for `MeshPhongMaterial` instead of `MeshStandardMaterial`? When I remember correctly, it was considered once to not support phong anymore in the upcoming renderer.

@mrdoob Do you have a preference here? I personally would start with a PBR material as a first lit material. Although phong would be easier to implement/port.

Reply:
> Any reasons why you have decided for MeshPhongMaterial instead of MeshStandardMaterial? 

It is a popular material and a more easier to implement... but I can start with `MeshStandardMaterial` I see no problem about it.

Reply:
Let's wait for @mrdoob's feedback. TBH, I'm not sure anymore if there was a conclusion about this topic and how it looked like.